spec/requests/summary_documents_api_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe 'POST /api/v1/summary_documents' do
+  include ActiveJob::TestHelper
+
+  scenario 'create a valid summary document activity' do
+    perform_enqueued_jobs do
```

Comment:
This (and the `include` above) isn't needed as the default adapter in test will run the jobs inline.